ipa-core/src/protocol/ipa_prf/shuffle/mod.rs

```diff
@@ -111,17 +145,90 @@ where
     }
 }
 
+// This function converts Attribution Outputs to an AdditiveShare needed for shuffle protocol
+pub fn attribution_outputs_to_shuffle_input<BK, TV, YS>(
```

Comment:
I think the naming is confusing and I do think it makes more sense to implement shared value (which implies additiveshare<sharedvalue>) for SecretSharedAttributionOutputs rather than merging BK and TV into a single Boolean Array. We might not even implement BooleanArrays that have the correct size needed.

Reply:
This entire part of the shuffling is going away anyway in favor of the sharded version which has a Shuffle-able trait

Reply:
I see so this is just a temporary change. Maybe mark it with a todo?

Reply:
yea that's pretty gross, but as @cberkhoff pointed out a necessary change due to shuffle not really supporting any other input except IPA event